MODULE.bazel

```diff
@@ -10,6 +10,7 @@ bazel_dep(name = "bazel_skylib", version = "1.7.1")
 bazel_dep(name = "platforms", version = "0.0.10")
 bazel_dep(name = "rules_apple", version = "3.8.0", repo_name = "build_bazel_rules_apple")
 bazel_dep(name = "rules_swift", version = "2.1.1", repo_name = "build_bazel_rules_swift")
+bazel_dep(name = "rules_cc", version = "0.0.16", repo_name = "build_bazel_rules_cc")
```

Comment:
Not sure whether we need to do anything else to make sure downstream packages handle `rules_cc` correctly

Reply:
No should be fine. IIRC rules_cc was split out of Bazel’s core in 7.0.